environment.yml

```diff
@@ -0,0 +1,19 @@
+name: pyuvsim
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - numpy>=1.15
+  - six>=1.11.0
+  - setuptools>=38.4.0
+  - scipy>=1.0.1
+  - PyYAML>=5.1
+  - mpi4py>=3.0.0
+  - astropy>=2.0
+  - pip
+  - psutil
+  - nose
+  - line_profiler
+  - pycodestyle
+  - h5py
+  - pyuvdata
```

Comment:
please add pytest-cov to the environment

Reply:
Done in ad351291a34d9d9d172605361b1e6df7c8ba2647